tests/Forms/DiffFieldTest.php

```diff
@@ -19,7 +19,7 @@ public function testScalarValuesAreDiffed()
         $diffField->setComparisonField($newField);
         $diffField->setValue('old');
 
-        $this->assertEquals('<ins>new</ins> <del>old</del>', $diffField->Value());
+        $this->assertRegExp('/^<ins>new<\/ins> *<del>old<\/del>$/', $diffField->Value());
```

Comment:
The sink somehow returns a slightly different number of spaces between those two tags. Don't really care why since it doesn't affect any functionality.

Reply:
```suggestion
        $this->assertRegExp('/^<ins>new<\/ins> +<del>old<\/del>$/', $diffField->Value());
```

One or more spaces instead of 0 or more?

Reply:
That's [what I'm trying to fix](https://travis-ci.com/github/silverstripe/cwp-recipe-kitchen-sink/jobs/504498555):
```
1) SilverStripe\VersionedAdmin\Tests\Forms\DiffFieldTest::testScalarValuesAreDiffed
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'<ins>new</ins> <del>old</del>'
+'<ins>new</ins><del>old</del>'

/home/travis/build/silverstripe/cwp-recipe-kitchen-sink/vendor/silverstripe/versioned-admin/tests/Forms/DiffFieldTest.php:22
/home/travis/build/silverstripe/cwp-recipe-kitchen-sink/vendor/sminnee/phpunit/phpunit:52
```

So there's a scenario where we get no space and a scenario where we got one space.

Maybe this would be more appropriate.

```suggestion
        $this->assertRegExp('/^<ins>new<\/ins> ?<del>old<\/del>$/', $diffField->Value());
```

Reply:
Ah. If we want to accept no spaces, then I think what you have is fine (0 or more)